pm_muairss_write/pm_muairss_write.xml

```diff
@@ -3,27 +3,47 @@
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
         pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
-        cp $structure input.cell &&
-        cp $params params.yaml &&
+        ln -s $structure input.cell &&
+        ln -s $params params.yaml &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            ln -s $castep_param castep.param &&
+        #end if
         pm-muairss -t w input.cell params.yaml &&
-        zip -r out_zip.zip $out_folder &&
-        find $out_folder > tree.txt
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        #if str($iscastep.iscastep_sel)=="yes":
+            zip -r out_zip1.zip \$out_folder params.yaml input.cell castep.param &&
+        #else if str($iscastep.iscastep_sel)=="no":
+            zip -r out_zip1.zip \$out_folder params.yaml input.cell &&
+        #end if
+        ln -s out_zip1.zip out_zip.zip &&
+        find \$out_folder > tree.txt
     ]]></command>
     <inputs>
         <param type="data" name="structure" format="cell" />
-        <param type="data" name="params" format="yaml" />
-        <param type="text" name="out_folder"/>
+        <param type="data" name="params" format="txt" />
+    <conditional name="iscastep">
+        <param type="select" value="no" name="iscastep_sel" label="Is the calculator castep?">
+            <option value="no">no</option>
+            <option value="yes">yes</option>
+        </param>   
+        <when value="yes">
+            <param type="data" name="castep_param" format="txt" />
+        </when>
+    </conditional>
     </inputs>
     <outputs>
-        <data label="$out_folder" name="out_zip" format="zip" from_work_dir="out_zip.zip"/>
+        <data label="muonated $structure.name-$params.name" name="out_zip" format="zip" from_work_dir="out_zip.zip"/>
```

Comment:
This is much better than before, but it will generate labels like `muonated Si.cell-Si-muairss-castep.yaml` which is a bit confusing to read with all the hyphenation. I suggest:
```suggestion
        <data label="muonated $structure.name using $params.name" name="out_zip" format="zip" from_work_dir="out_zip.zip"/>
```